File: applications/backend/api/models/city_airport.py
from pydantic import BaseModel

class CityBase(BaseModel):
    city_id: str
    city: str
    state_id: str
    lat: float
    lng: float
    ranking: int

class AirportServiceability(BaseModel):
    airport_serviceability_id: int
    city_id: str
    airport_id: str
    iata: str
    airport_name: str
    distance_miles: float
    hub: str

File: GUI/webapp/src/types/Search/cityAirport.ts
export interface SearchCityResponse {
    city_id: string;
    city: string;
    state_id: string;
    lat: number;
    lng: number;
    ranking: number
}

export interface SearchServiceableAirportsResponse {
  airport_serviceability_id: string;
  city_id: string;
  airport_id: string;
  iata: string;
  airport_name: string;
  distance_miles: number;
  hub: string;
}